src/scene/materials/standard-material.js

```diff
@@ -34,6 +34,12 @@ Object.assign(pc, function () {
      * @property {boolean} specularVertexColor Use mesh vertex colors for specular. If specularMap or are specularTint are set, they'll be multiplied by vertex colors.
      * @property {string} specularVertexColorChannel Vertex color channels to use for specular. Can be "r", "g", "b", "a", "rgb" or any swizzled combination.
      *
+     * @property {boolean} useAnisotropy Use anisotropic specular.
```

Comment:
Just wondering...if the value of anisotropy is zero, shouldn't the effect be disabled? And if that is so, do we really need `useAnisotropy`?

Reply:
This was also mentioned by the other guys - the isotropic specular result of GGX distribution differs from Phong / Blinn-Phong. Maybe it would make sense to make the bool switch to GGX specular with and add a non-anisotropic GGX shader chunk when anisotropy is 0?

Reply:
@mvaligursky @slimbuck opinions?

Reply:
If performance of GGX specular is not considerably worse than phong, perhaps we should switch standard shader to use it all the time (as few other engines did), and have a slider for anisotropic specular - and disable the chunk if slider is at 0. And also switch cubemaps to be GGX compatible.

Reply:
GGX isotropic specular (at anisotropy 0) looks different to the existing blinn-phong specular and adds a reasonable cost to the shader execution. So keeping the checkbox probably makes sense and means 'enable GGX specular' (which will also apply when we add the option of anisotropy direction map).

Reply:
OK, sounds good to me.

Reply:
I shall update the name of the checkbox

Reply:
I renamed it in the latest pushed changes